dashboards/node.libsonnet

```diff
@@ -35,9 +35,9 @@ local gauge = promgrafonnet.gauge;
           span=6,
           format='percent',
         )
-        .addTarget(prometheus.target('node_load1{%(nodeExporterSelector)s, instance="$instance"} * 100' % $._config, legendFormat='load 1m'))
-        .addTarget(prometheus.target('node_load5{%(nodeExporterSelector)s, instance="$instance"} * 100' % $._config, legendFormat='load 5m'))
-        .addTarget(prometheus.target('node_load15{%(nodeExporterSelector)s, instance="$instance"} * 100' % $._config, legendFormat='load 15m'));
+        .addTarget(prometheus.target('sum by (instance) (node_load1{%(nodeExporterSelector)s, instance="$instance"}) * 100' % $._config, legendFormat='load 1m'))
```

Comment:
we should just use the `max` value here, we're also already selecting by `instance`, so no need for the `by (instance)`